### ISSUE TITLE
Please, provile built firmware v0.38

### ISSUE THREAD

**Issue by nmaster2042** [MicroPhase/antsdr-fw-patch#19]
It's not easy to build lastest fimware for E200.

It would be very nice to provide built firmware v0.38 so we can easily update our devices.

**Comment by ohault** [comment]
@lone-boy @MicroPhase @black-pigeon Indeed from a customer point of view, MicroPhase should update both the documentation and firmware and drivers releases.

**Comment by 0x657373656C6C** [comment]
$500 for an e210 that is useless without microphase updating it

**Comment by lone-boy** [comment]
Hello, the firmware v0.38 has been updated in the v0.38 branch of github.
You can try git clone -b v0.38 --recursive https://github.com/MicroPhase/antsdr-fw-patch.git
@ohault @nmaster2042

**Comment by lone-boy** [comment]
The vivado tool version required by v0.38 is different from the previous one, and is the same as Plutosdr's V0.38
'export CROSS_COMPILE=arm-linux-gnueabihf- export VIVADO_SETTINGS=/opt/Xilinx/Vivado/2022.2/settings64.sh'

**Comment by ohault** [comment]
> Hello, the firmware v0.38 has been updated in the v0.38 branch of github. You can try git clone -b v0.38 --recursive https://github.com/MicroPhase/antsdr-fw-patch.git @ohault @nmaster2042

Hi @lone-boy, thank you. I was asking for your customers to also build and release on https://github.com/MicroPhase/antsdr-fw-patch/releases a v0.38 firmware.

**Comment by lone-boy** [comment]
Yes, we haven't released the V0.38 image yet, but we have updated it to v0.38 firmware when it leaves the factory, so if you bought it recently, I think it's the case, we will release this version as soon as possible.

**Comment by ohault** [comment]
@lone-boy Thank you.

**Comment by 0x657373656C6C** [comment]
> Yes, we haven't released the V0.38 image yet, but we have updated it to v0.38 firmware when it leaves the factory, so if you bought it recently, I think it's the case, we will release this version as soon as possible.

No, I bought mine from Mouser and it has 0.34, so something ready built to accommodate your distributors would be good.

**Comment by 0x657373656C6C** [comment]
> > Yes, we haven't released the V0.38 image yet, but we have updated it to v0.38 firmware when it leaves the factory, so if you bought it recently, I think it's the case, we will release this version as soon as possible.
> 
> No, I bought mine from Mouser < 2 weeks ago and it has 0.34, so something ready built to accommodate your distributors would be good.

**Comment by nmaster2042** [comment]
Same here: I bought mine on crowdsupply when it was founded so it came with 0.34.

It would be great to have a pre built upgradable 0.38 firmware.

**Comment by 0x657373656C6C** [comment]
It takes 4 days to compile does it, MicroPhase?

**Comment by lone-boy** [comment]
Thank you very much for your support during the crowdfunding stage. The source code of the V0.38 firmware is currently in the master branch. We still need to sort out the instructions, but here is the SD card firmware of the Pluto v0.38 firmware, you can try it first. It is recommended that you use the SD boot method to experience the v0.38 firmware.
[build_sdimg.zip](https://github.com/user-attachments/files/18061139/build_sdimg.zip)

**Comment by nmaster2042** [comment]
Thanks @lone-boy I will try this 0.38 firmware.
